### PR TITLE
chore(deps): loopback version update in services and packages

### PR DESCRIPTION
**Pull request by AnkurBansalSF**
## Description

- Updated versions of Loopback and Sourceloop packages.
- did changes related to deletedById (loopback4-soft-delete) in bearer-verifier (core) and component.ts (auth-service)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -1,96 +1,96 @@
 {
-  "name": "@sourceloop/core",
```
Comment (samarpan-b) [inline packages/core/package.json:2]: fix indentation

**Review by samarpan-b — CHANGES_REQUESTED** [review]

File: packages/core/package.json (hunk 1 above)
Comment (AnkurBansalSF) [reply packages/core/package.json]: Done

**Review by akshatdubeysf — APPROVED** [review]

File: packages/core/src/components/bearer-verifier/providers/facades-bearer-token-verify.provider.ts [hunk 2]
```diff
@@ -53,7 +60,12 @@ export class FacadesBearerTokenVerifyProvider
       ) {
         throw new HttpErrors.Unauthorized('PasswordExpiryError');
       }
-      return user;
+
+      if (this.authUserModel) {
```
Comment (samarpan-b) [inline packages/core/src/components/bearer-verifier/providers/facades-bearer-token-verify.provider.ts:64]: Why this check ?

File: packages/core/src/components/bearer-verifier/providers/facades-bearer-token-verify.provider.ts [hunk 3]
```diff
@@ -20,6 +25,8 @@ export class FacadesBearerTokenVerifyProvider
     @repository(RevokedTokenRepository)
     public revokedTokenRepository: RevokedTokenRepository,
     @inject(LOGGER.LOGGER_INJECT) private readonly logger: ILogger,
+    @inject(AuthenticationBindings.USER_MODEL, {optional: true})
```
Comment (samarpan-b) [inline packages/core/src/components/bearer-verifier/providers/facades-bearer-token-verify.provider.ts:28]: What is this new injection for ?

**Review by samarpan-b — CHANGES_REQUESTED** [review]

**Review by akshatdubeysf — APPROVED** [review]

**Pull request merged**